Cargo.lock

```diff
@@ -587,6 +646,17 @@ dependencies = [
  "winapi",
 ]
 
+[[package]]
+name = "parse_duration"
+version = "2.1.1"
+source = "registry+https://github.com/rust-lang/crates.io-index"
+checksum = "7037e5e93e0172a5a96874380bf73bc6ecef022e26fa25f2be26864d6b3ba95d"
+dependencies = [
+ "lazy_static",
+ "num",
```

Comment:
Hmm, I wonder if `parse_duration` really needs to pull in all of num + lazy_static.

Reply:
We already pull in once_cell which subsumes lazy_static. Looking at the `parse_duration` implementation too, it's 390 lines + tests and the `num` crate seems to primarily be used for bigint handling. What do you think about just copying over the code and simplifying?

Reply:
Sure, we probably need something much simpler. Seconds + minutes + hours, integers only, no exponents. Should be doable just with a regular expression.

Reply:
I added a custom parser, and removed this dependency.